src/blocks/ProductList/product/Component.tsx

```diff
@@ -1,32 +1,136 @@
   return (
-    <li className="container mt-16">
-      <div className="max-w-[48rem]">
-        {product.logo &&
-          <Media
-            imgClassName='h-32 w-32 border border-border rounded-[0.8rem]'
-            resource={product.logo}
-          />
-        }
-        <br />
-        Name: {product.name}
-        <br />
-        Description: {product.description ?? "No description"}
-        <br />
-        Link: {product.link ?? "No link"}
-      </div>
-    </li>
+    <>
+      <Card className='h-full flex flex-col'>
+        <CardHeader>
+          <div className='flex flex-col md:flex-row justify-between'>
+            <div className='flex items-center justify-center'>
+              <Media
+                imgClassName='w-12 h-12 border border-border rounded-full'
+                resource={product.logo}
+                alt={`${product.name} logo`}
+              />
+              <H2 className='mx-2 pb-0 flex-wrap'>
+                {product.name}
+              </H2>
+            </div>
+            <div className='flex items-center justify-center '>
+              <H4 className='pb-0'>
+                {product.producedIn?.map((c: Country | number) => {
+                  if (typeof c !== 'number') {
+                    return <p key={c.id}>{c.name}</p>
+                  }
+                }
+                )}
+              </H4>
+            </div>
+          </div>
+        </CardHeader>
+
+        <div className='flex flex-col justify-between h-full gap-2'>
+          <CardDescription className='text-ellipsis overflow-hidden h-fit p-6' >
+            {product.description}
+          </CardDescription>
+          <div className='flex flex-col'>
+            <CardContent className='flex flex-col justify-end text-xs gap-2'>
+
+              {hasCategory && (
+                <div>
+                  <div className='flex items-center'>
+                    <p className='font-semibold text-gray-800 dark:text-gray-300'>Category:</p>
+                  </div>
+                  <div className='flex gap-2 flex-wrap'>
+                    {product.categories?.map((category: Category | number) => {
+                      if (category !== undefined && typeof category !== 'number') {
+                        return (
+                          <p key={category.id} className='text-blue-500 font-medium border-2 border-blue-500 p-1 rounded'>
+                            {category.name}
+                          </p>
+                        );
+                      }
+                      return null;
+                    })}
+                  </div>
+                </div>
+              )}
+
+              {hasSubcategory && (
+                <div>
+                  <div className='flex items-center'>
+                    <p className='font-semibold text-gray-800 dark:text-gray-300'>Subcategory:</p>
+                  </div>
+                  <div className='flex gap-2 flex-wrap'>
```

Comment:
![img_yXgapG3S@2x](https://github.com/user-attachments/assets/59b3c7b6-e8e9-465b-8f84-1d573d05ace4)
I think we could use more spacing here.